### PR TITLE
Updated the year in the LICENSE file.

### PR DESCRIPTION
**Pull request by FenrirWillow**
We might as well since it is 2018 for almost 6 months now 😉

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Dubit
+Copyright (c) 2018 Dubit
```
Comment (Rollo-Liu) [inline LICENSE]: Maybe `2017-2018` Dubit Limited

Comment (Rollo-Liu) [reply LICENSE]: or XR Games :trollface:

**Comment by FenrirWillow** [comment]
You do not include every year in the copy right notice, only the current year (or last year that the copyright was active).

@pmead I don't know what is our practice in terms of Copyright holder signature, I guess it should be "Dubit Limited" since that is the actual corporate entity.

**Comment by Rollo-Liu** [comment]
Wrong. You should have a ranged year for any content that still included and was in the last active year.

Do we still have the content from 2017? Yes we have. So it's 2017-2018.

**Comment by FenrirWillow** [comment]
Made that year change, also included Ltd. in the name of the company.

**Pull request merged**